srcpkgs/strawberry/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'strawberry'
+pkgname=strawberry
+version=0.4.2
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-host-tools qt5-devel protobuf-c"
+# maybe also liblastfm, but that depends on qt4 right now
+makedepends="chromaprint-devel boost-devel gst-plugins-base1-devel
+ sqlite-devel protobuf-devel qt5-devel qt5-x11extras-devel qt5-plugin-tds
+ qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc
+ libplist-devel libusbmuxd-devel libmtp-devel libcdio-devel taglib-devel
+ libgpod-devel pulseaudio-devel alsa-lib-devel"
+depends="qt5-plugin-sqlite desktop-file-utils"
+short_desc="An audio player and music collection organizer"
+maintainer="q66 <daniel@octaforge.org>"
+license="GPL-3"
```

Comment:
Use the SPDX-Identifier with `-or-later` or `-only`. Judging by the few files I checked it should `GPL-3.0-or-later`